Week2/Interface example/InterfaceExample.java

```diff
@@ -0,0 +1,12 @@
+public class InterfaceExample {
+
+    public static void main(String[] args) {
+        
+        Bike bike= new RoyalEnfield();
+        bike.Company();
+        bike.SpeedLimit();
+        bike = new Honda();

```

Comment:
Please give input as speed and display message as per requirement

Reply:
Fixed.